03/app.js

```diff
@@ -0,0 +1,33 @@
+const textClicked = function() {
```

Comment:
W tym zadaniu faktycznie trzeba wyszukać wszystkie elementy (przy pomocy `.querySelectorAll()`) i wykorzystać pętle, aby nie powielać kodu. W tym przypadku wystarczy jedna funkcja, która jest przypisana do "każdego buttona".

Jednak w jej wnętrzem możesz wykorzystać `this`, aby odwołać się do konkretnego przycisku.